openrtb_ext/imp_rubicon.go

```diff
@@ -15,6 +15,7 @@ type ExtImpRubicon struct {
 	Visitor          json.RawMessage    `json:"visitor,omitempty"`
 	Video            rubiconVideoParams `json:"video"`
 	Debug            impExtRubiconDebug `json:"debug,omitempty"`
+	PChain           string             `json:"pchain,omitempty"`
```

Comment:
should create docs PR to include description for this `pchain` bidder param ([rubicon.md](https://github.com/prebid/prebid.github.io/blob/8f9ef40d8e53204004ff7e327460a0d6a67b2d22/dev-docs/bidders/rubicon.md#L32-L45))

Reply:
I'm actually pushing to get rid of this deprecated param entirely, but went ahead and added it to the docs so this PR can be merged. We'll remove it as a separate effort if I get the ok . https://github.com/prebid/prebid.github.io/pull/4397